checker/lib/README.developers

```diff
@@ -0,0 +1,11 @@
+Please read the README file in this directory first, as it is the file
+that is shipped in the distribution and is intended for users of the
+Checker Framework.
+
+When adding a .jar file to this directory, update the list in README and also
```

Comment:
Are there instructions for how to create a new annotated JDK file?
If not, is this file the right place to add them?